judgels-frontends/raphael/src/routes/courses/courses/single/chapters/single/submissions/ChapterSubmissionsPage/ChapterSubmissionsPage.jsx

```diff
@@ -20,7 +21,11 @@ import * as chapterSubmissionActions from '../modules/chapterSubmissionActions';
 export class ChapterSubmissionsPage extends Component {
   static PAGE_SIZE = 20;
 
-  state;
+  state = {
+    isDialogOpen: false,
```

Comment:
Can we call these `isSubmissionImageDialogOpen`, `submissionDialogTitle`, `submissionImageUrl`, `toggleSubmissionImageDialog`, everywhere?